### PR TITLE
go only, fixed some stuff

### PR DESCRIPTION
**Pull request by WorkHorseIndustries**
Remove dependency on daddye/vips which used an underlying c library.  I'm pushing this into dev to test out performance. 

feel free to ignore `/vendor` folder.

### CODE REVIEW

**Comment by WorkHorseIndustries** [comment]
@vokal/systems

**Comment by foresmac** [comment]
There's a lot of weird stuff that looks like it shouldn't be in the vendor folder... like vip itself.

**Comment by WorkHorseIndustries** [comment]
@foresmac so to handle inter-package dependencies within the same module, you set up a remote source, then refer to THAT instead of using relative paths. See http://stackoverflow.com/questions/15049903/how-to-use-custom-packages-in-golang#15051192

**Comment by foresmac** [comment]
But why would the test samples be a separate `go get`-able package?

**Comment by foresmac** [comment]
If you're not using the `vips` binding then you don't need to install that, either. And you could skip the `bootstrap.sh` script because that's all for installing some or all of that stuff. You could probably drop a lot of other stuff installed in the container as well.

**Comment by WorkHorseIndustries** [comment]
You're right, but I didn't change any tests here. I can open an issue to fix that.  However the different packages are referenced in places other than the test. It probably should be re-organized under a single package, but eh, I can do that later.

**Comment by WorkHorseIndustries** [comment]
Re: Drone stuff
Cool, thanks.

**Comment by foresmac** [comment]
I would probably take everything extra out of the container and add it in as needed; but if you're only using golang, then there's probably nothing you need to install.

File: .drone.yml [hunk 1]
```diff
@@ -5,16 +5,6 @@ build:
...
-        - apt-get update -qq
-        - apt-get install -y libvips38 libxml2-dev automake build-essential git gobject-introspection libglib2.0-dev libjpeg62-turbo-dev libpng12-dev gtk-doc-tools
-        - git clone https://github.com/jcupitt/libvips.git
-        - cd libvips
-        - ./bootstrap.sh
-        - ./configure --enable-debug=no --without-python --without-fftw --without-libexif --without-libgf --without-little-cms --without-orc --without-pango --prefix=/usr
-        - make
-        - make install
-        - ldconfig
-        - cd ..
```
Comment (foresmac) [inline .drone.yml:17]: 👍

**Comment by WorkHorseIndustries** [comment]
werd

**Comment by foresmac** [comment]
The tests are failing because of differences in the way fractional pixel sizes were calculated using libvips vs what's in the code now, btw.

**Comment by WorkHorseIndustries** [comment]
Yeah, I know, it sucks.

**Comment by WorkHorseIndustries** [comment]
@foresmac @vokal/systems 

It's like old times Foresman, you saying stuff, me saying stuff, us saying stuff together.

**Comment by projectweekend** [comment]
LGTM

**Comment by foresmac** [comment]
I'm sure it's probably fine—I mean, hey, the tests pass!—but I'm not going to look through all 107 files! ;)

**Comment by foresmac** [comment]
SCOOPS

**Comment by WorkHorseIndustries** [comment]
SCOOP, SCOOP, SCOOP-A-LOOP.

**Comment by scottferg** [comment]
Scoops bros

**Comment by foresmac** [comment]
🍨 🍨 🍨 🍨 🍨 🍨 🍨

**Pull request merged**